cmd/new/new.go

```diff
@@ -133,6 +133,11 @@ func execute(cmd *cobra.Command, args []string) error {
 		return nil
 	}
 
+	_, err := pct.Get(localTemplateCache, selectedTemplate)
+	if err != nil {
+		log.Fatal().Msgf("Couldn't find an installed template that matches '%s'", selectedTemplate)
```

Comment:
Could we use the existing error message like [this](https://golang.org/pkg/fmt/#example_Errorf)? 
```suggestion
		log.Fatal().Msgf(err.Error)
```

Reply:
Good catch! I moved the custom error message to the pct get method but forgot to remove it from the new command.